.github/workflows/ci.yml

```diff
@@ -77,3 +77,26 @@ jobs:
       run: cargo test --verbose --features x509-parser
     - name: Run cargo doc
       run: cargo doc --all --all-features
+
+  coverage:
+    name: Measure coverage
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+      - name: Install cargo-llvm-cov
+        uses: taiki-e/install-action@cargo-llvm-cov
+      - name: Install rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          components: llvm-tools
+      - name: Measure coverage
```

Comment:
IIRC CodeCov is pretty flaky unless you provide a token.

Reply:
Added a token.